pkg/cloud/aws/actuators/scope.go

```diff
@@ -76,6 +77,7 @@ func NewScope(params ScopeParams) (*Scope, error) {
 		ClusterClient: clusterClient,
 		ClusterConfig: clusterConfig,
 		ClusterStatus: clusterStatus,
+		Logger:        params.Logger,
```

Comment:
We could use the information we have in the cluster and machine Scope to prefix WithName with apigroup + /namespace + [/machine/<id> or / cluster/<id>] and avoid the repetition.

Reply:
Actually, I really like this idea, except I would say:
```
apigroup + /namespace + /cluster/ [ + machine/ ], since the machine will have a cluster (at least with the current code).

Reply:
Even better!

Reply:
yeeee that sounds great